app/Services/user.service.ts

```diff
@@ -0,0 +1,14 @@
+import User from "App/Models/Auth";
+
+export default class UsersService {
+
+    public static async registerUser(payload: Object) {
+
+        const user = await User.create(payload);
+        return user.toJSON();
```

Comment:
Just returning the user object will do the job. Serialising it to json won't be necessary as it does it automatically.